kong/router/utils.lua

```diff
@@ -234,6 +234,131 @@ local function get_service_info(service)
          service_path
 end
 
+local phonehome_statistics
+do
+  local reports = require("kong.reports")
+  local nkeys = require("table.nkeys")
+  local log = ngx.log
+  local ERR = ngx.ERR
+
+  local byte = string.byte
```

Comment:
It seems we already have `local byte = string.byte` in this module, please check it.

Reply:
If that is true lint should warn about it.

Reply:
See https://github.com/Kong/kong/blob/f416c76e88cb1f1d189c84d2f0b6cf9404c4435d/kong/router/utils.lua#L9